tests/core/manager/test_response_formatters.py

```diff
@@ -4,6 +4,9 @@
 from eth_utils.toolz import (
     identity,
 )
+from toolz import (
+    merge,
```

Comment:
this should be imported via `eth_utils` like identity above. There's some logic in eth-utils around using toolz or cytoolz

Reply:
ah, yep!